elliottlib/cli/find_bugs_cli.py

```diff
@@ -220,16 +220,52 @@ def _filter_bugs(bugs):  # returns a list of bugs that should be processed
     # Otherwise we don't need to sweep bugs at all.
     if not (into_default_advisories or default_advisory_type):
         return
-    impetus_bugs = {}  # key is impetus ("rpm", "image", "extras"), value is a set of bug IDs.
-    # @lmeyer: simple and stupid would still be keeping the logic in python, possibly with config flags for branched logic. until that logic becomes too ugly to keep in python, i suppose..
+
+    # key is impetus ("rpm", "image", "extras"), value is a set of bug IDs.
+    impetus_bugs = {
+        "rpm": set(),
+        "image": set(),
+        "extras": set()
+    }
+
+    # @lmeyer: simple and stupid would still be keeping the logic in python,
+    # possibly with config flags for branched logic.
+    # until that logic becomes too ugly to keep in python, i suppose..
     if major_version < 4:  # for 3.x, all bugs should go to the rpm advisory
         impetus_bugs["rpm"] = set(bugs)
     else:  # for 4.x
-        # optional operators bugs should be swept to the "extras" advisory, while other bugs should be swept to "image" advisory.
-        # a way to identify operator-related bugs is by its "Component" value. temporarily hardcode here until we need to move it to ocp-build-data.
+        # sweep rpm cve trackers into "rpm" advisory
+        if mode == 'sweep' and cve_trackers:
```

Comment:
If `cve_trackers` is false, we still need to sweep "normal" bugs here. Just CVE trackers are not swept.

Reply:
I asked @sosiouxme about this part, specifically attaching normal (non-cve) bugs to rpm advisory and from what I understood is, we cannot rely for every bug to have the rpm identification logic (`component:` or `-container` if we're using that). I could be wrong, let me know and I'd be happy to change this logic.